plugins/CoreHome/vue/src/PeriodSelector/PeriodSelector.vue

```diff
@@ -250,7 +250,7 @@ export default defineComponent({
   },
   mounted() {
     Matomo.on('hidePeriodSelector', () => {
-      window.$(this.$refs.root as HTMLElement).hide();
+      window.$(this.$refs.root as HTMLElement).parent().hide();
```

Comment:
maybe hide the parent only if it has [vue-entry]? this way if PeriodSelector is used directly in a Vue template, an element in the parent component wont be accidentally hidden

Reply:
@diosmosis ah I found the issue case, the parent doesn't have [vue-entry], but should hide.

Reply:
ok

Reply:
Doesn't the parent div always have the id `periodString` in case it should be hidden?

Reply:
@sgiehl we probably can check if the id has `periodString` good point.